build_tools/kokoro/gcp_ubuntu/cmake/riscv/riscv64-linux/build_kokoro.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
```

Comment:
The naming convention we follow for these is build_system/os/architecture, so I think this should be `build_tools/kokoro/gcp_ubuntu/cmake/linux/riscv64/build_kokoro.sh`

Reply:
Done and files moved.